Simplify KubernetesPodOperator Local Instructions - `docker-desktop` kubeconfig

alex-astronomer opened the pull request:
Instead of manually copying part of the contents of the `kubeconfig` file, we can run `kubectl` commands which will accomplish this automatically without having to manually copy parts of files into another.

kentdanas reviewed (approved):
Looks good, thanks @alex-astronomer!

alex-astronomer commented:
Made some small changes, so re-requesting review.  Removed the certificate step since the `--raw` flag will get all of the certificate data, and also adjusted the next step to reflect the changes that might be needed as optional troubleshooting.  Maybe this step could be changed to a "Tip" or "Insight" box or something similar that I've seen in other documentation?

kentdanas commented on learn/kubepod-operator.md:
````diff
@@ -122,35 +122,39 @@ The latest versions of Docker for Windows and Mac let you run a single node Kube
+    client-key-data: <client-key-data>
+    ```
+
+2. If facing issues connecting, check the server configuration in the `kubeconfig` file. If `server: https://localhost:6445` is present, change to `server: https://kubernetes.docker.internal:6443` to identify the localhost running Kubernetes Pods. If this doesn't work, try `server: https://host.docker.internal:6445`.
````
```suggestion
2. If you have issues connecting, check the server configuration in the `kubeconfig` file. If `server: https://localhost:6445` is present, change to `server: https://kubernetes.docker.internal:6443` to identify the localhost running Kubernetes Pods. If this doesn't work, try `server: https://host.docker.internal:6445`.
```

kentdanas reviewed (approved):
Thanks @alex-astronomer. One small language suggestion, otherwise LGTM! I would prefer keeping the notes in number format like you have them, since there are multiple. Having long or multiple tip boxes in a row gets distracting :)
